pkg/controller/volume/attachdetach/metrics/metrics_test.go

```diff
@@ -71,6 +72,7 @@ func TestVolumesInUseMetricCollection(t *testing.T) {
 			Name:      "metric-test-pvc",
 			Namespace: "metric-test",
 			UID:       "metric-test-pvc-1",
+			Tenant: metav1.TenantDefault,
```

Comment:
you need to run "make update" before checking in, as the format does not look right.

Reply:
fixed